ztunnel/build.rs

```diff
@@ -84,3 +88,11 @@ fn main() -> Result<(), anyhow::Error> {
     );
     Ok(())
 }
+
+fn workspace_dir() -> PathBuf {
+    let mut dir = std::env::current_dir().unwrap();
```

Comment:
This seems to work, although not sure if there is a better way to get the workspace dir.